packages/cli/src/cmds/validator/handler.ts

```diff
@@ -220,7 +220,7 @@ function getProposerConfigFromArgs(
   }: {persistedKeysBackend: IPersistedKeysBackend; accountPaths: {proposerDir: string}}
 ): ValidatorProposerConfig {
   const defaultConfig = {
-    graffiti: args.graffiti ?? getDefaultGraffiti(),
+    graffiti: args.graffiti ?? "",
```

Comment:
default graffiti should be `undefined`, not an empty string. A user setting `--graffiti '"` must result in an empty graffiti in the block